Dockerfile

```diff
@@ -100,7 +106,8 @@ RUN apk add --no-cache \
     tzdata \
     unzip \
     which \
-    font-noto-cjk
+    font-noto-cjk \
+    nodejs=${NODEJS_VERSION}
```

Comment:
It's probably best to just use the Node.js version that Alpine provides.

Reply:
👌🏻